app/src/main/java/helium314/keyboard/latin/utils/ToolbarUtils.kt

```diff
@@ -36,12 +36,21 @@ fun createToolbarKey(context: Context, keyboardAttr: TypedArray, key: ToolbarKey
 
 fun getCodeForToolbarKey(key: ToolbarKey) = when (key) {
```

Comment:
What exactly did you change here? Re-arranging makes the diff very hard to read.

Reply:
Same for the other changes below. It's all in the [add paste key](https://github.com/Helium314/HeliBoard/pull/945/commits/106314b6b00f03b72ec73cac07879fd2b24a35a4) commit, it is really only re-arranging and adding the paste key?

Reply:
Yes.